pyomo/contrib/pyros/uncertainty_sets.py

```diff
@@ -767,31 +767,58 @@ def parameter_bounds(self):
 
     def set_as_constraint(self, uncertain_params, **kwargs):
```

Comment:
Can you remove `**kwargs` here? Doesn't look like it is being used at all in this method.

Reply:
@blnicho I've noticed that removing the `**kwargs` may result in a `TypeError` during the course of a PyROS solve attempt in which the uncertainty set is an `AxisAlignedEllipsoidalSet`. In particular, this issue arises from [lines 107--109 of uncertainty_sets.py](https://github.com/shermanjasonaf/pyomo/blob/280e4c76e49e02f613444a8d33c8f381e90c825b/pyomo/contrib/pyros/uncertainty_sets.py#L107), and [lines 30--32 of separation_problem_methods.py](https://github.com/shermanjasonaf/pyomo/blob/280e4c76e49e02f613444a8d33c8f381e90c825b/pyomo/contrib/pyros/separation_problem_methods.py#L30), in which the `set_as_constraint` method is invoked generically for any uncertainty set, without accounting for the differences between the arguments accepted by the `set_as_constraint` methods for different `UncertaintySet` subclasses. Currently, the presence of the `**kwargs` argument(s) seems to "guard" against this issue. For now, I've restored the `**kwargs` argument and added a test case in a subsequent commit.

Reply:
I think the correct solution is to update the signature here to be:
```
def set_as_constraint(self, uncertain_params, model, config):
```
to match the other APIs.

Reply:
See subsequent commits